test/sample/MemberGetBalanceSample.spec.js

```diff
@@ -10,7 +10,7 @@ import MemberGetBalanceSample from '../../src/sample/MemberGetBalanceSample';
 describe('MemberGetBalanceSample test', () => {
     it('Should run the sample', async () => {
         const member = await CreateMemberSample();
-	await LinkMemberAndBankSample(member);
+	    await LinkMemberAndBankSample(member);
```

Comment:
(in this github web-diff view, both the old and the new indentation look off.)

Reply:
Yeah I think I messed up the indent the first time I tried to fix it, but it looks ok now.